src/components/court-order-poa/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/court-order-poa",
-  "version": "2.1.3",
+  "version": "3.0.0",
```

Comment:
Don't change this. It is set by step 14 in the readme (which is done by a repo admin).

Reply:
Got it.  Thanks @severinbeauvais